cmd/trade.go

```diff
@@ -31,6 +32,8 @@ var tradeCmd = &cobra.Command{
 	Example: tradeExamples,
 }
 
+var l *logger.BasicLogger
```

Comment:
we should not have *any* global vars -- if you need this in the logPanic() method then logPanic can take in the logger interface